### PR TITLE
Create adamstrange.yml

### PR DESCRIPTION
**Pull request by TheLightShadow**
https://www.planetminecraft.com/skin/adam-strange---dc-comics/ - Skin

https://vignette.wikia.nocookie.net/braveandbold/images/0/0b/Adam_Strange_10.jpg/revision/latest?cb=20110918203535 - Jetpack / Jetpack Fuel

https://www.youtube.com/watch?v=r3ZD9kC6HCA - 1:31 - 1:41 - Tp pos

https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTAZvrDN_vhMJY9Tq_L2x9-zVEcsPLsgtbN7djxzaOyVgRhqrSc - Energy Pistol

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/chars/adamstrange.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  val: eyJ0aW1lc3RhbXAiOjE1MjE3ODUyMzUxODMsInByb2ZpbGVJZCI6ImQxZmJhYWI3YmRlMzQ0MDU4MjgyOWZjMTVkOTAyMDlkIiwicHJvZmlsZU5hbWUiOiJUaGVEYXJrQXVyYV8iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzk5NjUzZjNiMTY2ZTU3NmQ5ZDJkNjJkNTM0MWQ5ZTgyNjg5NTkyMWExNjA0YWQzYTFlOTQwZjVhNmUzYjU5In19fQ==
+items:
+- /give <plr> minecraft:fireworks 64 name:&8Jetpack Fuel
+- /give <plr> minecraft:elytra 1 70:1 34:5 name:&6Jetpack
+- /shot give <plr> energypistol
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/adamstrange.yml]: we dont have an energy pistol

File: CommandHelper/LocalPackages/commands/chars/adamstrange.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+permissions:
+  superherocity:
+  - essentials.tppos
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/adamstrange.yml]: tppos is denied, looks like happenstance circumstance that he was TPed, not his own power

File: CommandHelper/LocalPackages/commands/chars/adamstrange.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+powers:
+- /items
+- /tppos <x> <y> <z>
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/adamstrange.yml]: tppos is denied, looks like happenstance circumstance that he was TPed, not his own power

File: CommandHelper/LocalPackages/commands/chars/adamstrange.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  val: eyJ0aW1lc3RhbXAiOjE1MjE3ODUyMzUxODMsInByb2ZpbGVJZCI6ImQxZmJhYWI3YmRlMzQ0MDU4MjgyOWZjMTVkOTAyMDlkIiwicHJvZmlsZU5hbWUiOiJUaGVEYXJrQXVyYV8iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzk5NjUzZjNiMTY2ZTU3NmQ5ZDJkNjJkNTM0MWQ5ZTgyNjg5NTkyMWExNjA0YWQzYTFlOTQwZjVhNmUzYjU5In19fQ==
+items:
+- /give <plr> minecraft:fireworks 64 name:&8Jetpack Fuel
+- /give <plr> minecraft:elytra 1 70:1 34:5 name:&6Jetpack
+- /shot give <plr> energypistol
+permissions:
+  superherocity:
+  - essentials.tppos
+powers:
+- /items
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/adamstrange.yml]: this is not a power

File: CommandHelper/LocalPackages/commands/chars/adamstrange.yml (hunk 1 above)
Comment (TheLightShadow) [reply CommandHelper/LocalPackages/commands/chars/adamstrange.yml]: https://github.com/malonnnn/ServerConfig2/blob/master/CrackShot/weapons/energypistol.yml - Energy Pistol.

**Pull request closed without merging**